account_invoice_tax_required/tests/test_account_invoice_tax_required.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# Copyright <YEAR(S)> <AUTHOR(S)>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+from openerp import exceptions
+
+
+class TestAccountInvoiceTaxRequired(TransactionCase):
+
+    def setUp(self, *args, **kwargs):
+        super(TestAccountInvoiceTaxRequired, self).setUp(*args, **kwargs)
+        self.main_company = self.env.ref('base.main_company')
+        res_users_account_manager = self.env.ref(
```

Comment:
There's no need to create a user for this. This test can be executed with admin.